src/Project/Project.router.ts

```diff
@@ -83,7 +83,7 @@ export class ProjectRouter extends Router {
      * Update all projects with coords to null
      */
     this.router.delete(
-      '/projects/coords/:coords',
+      '/projects/:coords/coords',
```

Comment:
Please use consistent URLs, all other endpoints use the `/projects/:id` prefix. This would make the API inconsistent and hard to both understand and maintain

Reply:
I have made this changes because @nicosantangelo  asked me to do it. Before the last commit it was this way, please discuss about which one you prefer and I will implement it!

Reply:
I share the sentiment, the problem here (I think) is that this entire interaction was not planned beforehand and the explorer does not have the id available to make requests. I'm all for using :id here but I don't know what that change'd imply

Reply:
I don't understand what you mean there. Are you suggesting creating a new router? Or I should make an API call to get all the projects with IDs and then do a call to removes the coords of all IDs? (That will be 2 API calls instead of 1, increasing the use of the server)

Reply:
Ok, I asked for some context offline and I got that this is so because you don't have the project id. 

If that is the case, I recommend to seek for consistency and use a proper `/project_by_coord/:coords/coords` url and call it a day, to not mess up with the routes.

I also want to raise the concern for future developments: this kind of design considerations should arise in early stages of thinking, designing and reaching consensus of the features. Not in pull requests. Because this process is not beneficial for anybody, and the end solution ends up being far from ideal, breaking conventions and looking like a "patch to make it work".

Reply:
Done! Ok. Next time I will do a meeting to ensure that we are following the same path